### DISCUSSION TITLE
Jellyseerr Lidarr support from Overseerr already implemented branch and then Import music from big streaming platfroms

### DISCUSSION THREAD

**Discussion by yru4m0** [seerr-team/seerr#1143] [Ideas]
I merged on a fork this [Overseer branch](https://github.com/ano0002/Overseerr-Lidarr-Integration) into develop on my [branch](https://github.com/yru4m0/jellyseerr/tree/feature-lidarr-integration) and it is working pretty but still can be better.

So the goal is to have Lidarr support into Jellyseer and then, being able to import music from big streaming platforms

The idea would be to import music from big streaming platforms using a website like [TuneMyMusic](https://www.tunemymusic.com/) to lidarr where you can backup a csv of all your music (for "free"). 
![Screenshot from 2024-11-25 20-06-26](https://github.com/user-attachments/assets/53647e60-d7fc-4733-b14a-e103c0287128)

We would have an import button that send a file to the server, then we parse this csv to import the music to Lidarr with the merged branch when it is working and ta-da!
I'm thinking to add like an advanced option to tell the column name to read and if it is correspond to artists or albums.

This feature could be interesting for all the people that doesn't changes because "I got 3000 liked songs that I will never import on Lidarr".

I am happy to work on this implementation and don't hesitate to tell me if you have suggestions or if you want to work on it too.

**Comment by gauthier-th** [comment]
You should take a look at #96 and talk with @0-Pierre, he's also working on it

**Reply by yru4m0** [reply]
Thank you I haven't seen it!
